documentation/asciidoc/computers/remote-access/vnc.adoc

```diff
@@ -1,123 +1,85 @@
 [[vnc]]
 == Virtual Network Computing (VNC)
-:experimental:
 
-Sometimes it is not convenient to work directly on the Raspberry Pi. Maybe you would like to work on it from another device by remote control.
+Sometimes it is not convenient to physically work with a device. Virtual Network Computing (VNC) allows you to control the desktop of one device from another.
 
-VNC is a graphical desktop sharing system that allows you to remotely control the desktop interface of one computer (running VNC Server) from another computer or mobile device (running VNC Viewer). VNC Viewer transmits the keyboard and either mouse or touch events to VNC Server, and receives updates to the screen in return.
+VNC relies upon a client and a server. The client runs on a device you can physically interact with, such as a personal laptop, desktop, tablet, or phone. The server runs on your Raspberry Pi.
+When you use VNC, the client transmits keyboard and mouse events to the server. The server executes those events on your Raspberry Pi, and returns screen updates to the client.
 
-You will see the desktop of the Raspberry Pi inside a window on your computer or mobile device. You'll be able to control it as though you were working on the Raspberry Pi itself.
+The VNC client displays the desktop of your Raspberry Pi in a window. You can interact with the desktop as though you were working on the Raspberry Pi itself.
 
-VNC Connect from RealVNC is included with Raspberry Pi OS. It consists of both VNC Server, which allows you to control your Raspberry Pi remotely, and VNC Viewer, which allows you to control desktop computers remotely from your Raspberry Pi should you want to.
+Raspberry Pi OS includes https://github.com/any1/wayvnc[wayvnc]. This provides a VNC server that you can enable in your device preferences.
 
-You must enable VNC Server before you can use it. By default, VNC Server gives you remote access to the graphical desktop that is running on your Raspberry Pi, as though you were sitting in front of it.
+Before you can use VNC on your Raspberry Pi, you must enable the VNC server.
 
-However, you can also use VNC Server to gain graphical remote access to your Raspberry Pi if it is headless or not running a graphical desktop. For more information on this, see *Creating a virtual desktop*, further below.
+NOTE: Previous versions of Raspberry Pi OS supported client connections from RealVNC. In version _Bookworm_ and above, Raspberry Pi OS no longer supports client connections from RealVNC. Instead, you can connect using TigerVNC.
 
-=== Installing VNC on Raspberry Pi
+=== Install VNC on Raspberry Pi
 
-VNC is already installed on the full Raspberry Pi OS image, and can be installed via `Recommended Software` from the `Preferences` menu on other versions.
+Raspberry Pi OS comes with WayVNC already installed. If you run a version of Raspberry Pi OS that does not include WayVNC, you can:
```

Comment:
If Raspberry Pi OS comes with WayVNC already installed, I don't think that we need to provide instructions on how to install it?

Reply:
Yup. The installation steps were a holdover from the days when it didn't. The VNC server comes pre-installed on both desktop and lite distros, don't need to step through installation.

Reply:
The old docs included install information, so I preserved it here. Apparently the Lite distribution didn't include the old VNC stack; I haven't had enough SD cards to verify if Bookworm includes VNC stuff in its lite distro (but I'm assuming it doesn't, since it doesn't seem essential).

Reply:
Scratch that; GitHub didn't show me Al's comment earlier. I'll remove the installation instructions then! Fewer words better.